src/Aquifer.Jobs/appsettings.json

```diff
@@ -0,0 +1,40 @@
+{
+    "Analytics": {
+        "ApiManagementResourceId": "/subscriptions/dacfa945-984c-4708-9f2d-7d8358f223d5/resourceGroups/BiblioNexus-Global/providers/Microsoft.ApiManagement/service/aquifer-api-management",
+        "ApiRequestStatsTableName": "ApiRequestStatsQa",
+        "AppInsightsResourceId": "/subscriptions/dacfa945-984c-4708-9f2d-7d8358f223d5/resourceGroups/BiblioNexus-Dev/providers/microsoft.insights/components/appinsights-qa",
+        "CustomEventsTableName": "CustomEventsQa",
+        "HoursBetweenRuns": 1,
+        "PageViewsTableName": "PageViewsQa",
+        "StorageAccountUri": "https://aquiferstoragedev.table.core.windows.net"
+    },
+    "Aquifer":{
+        "AdminBaseUri": "Base URL of Aquifer API",
+        "ApiBaseUri": "Base URL of the Aquifer Admin"
+    },
+    "ConnectionStrings": {
+        "ApplicationInsights": "Application insights connection string. Get from Azure instance.",
+        "AzureStorageAccount": "Storage connection string",
+        "BiblioNexusDb": "DB connection string"
+    },
+    "Email": {
+        "SubjectPrefix": "[Test Email - Local] ",
+        "Marketing": { 
+            "Address": "Email address that the new content email is coming from",
+            "Name": "Name that the new content email is under when email is sent",
+            "ResourceLink": "Link to resources in new content email template"
```

Comment:
Instead of leaving in the examples we should make these all empty instead.  See https://github.com/BiblioNexusStudio/aquifer-server/blob/master/src/Aquifer.API/appsettings.json as an example.